### PR TITLE
Add splitShuntAdmittance in threeWindingsTransformer. Only one flag "twtSplitShuntAdmittance" for twoWindings and threeWindingsTransformers

### PR DESCRIPTION
**Pull request by marqueslanauja**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
There is no possibility to split the shunt admittance in `threeWindingsTransformers` on Load Flow execution.

**What is the new behavior (if this is a feature change)?**
A new Load Flow parameter has been added so it is possible to configure the shunt admittance split in `threeWindingsTransformer `on Load Flow execution.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

`SplitShuntAdmittance `has not been extended to the ampl-converter package as this package is not using the current IIDM API of the `threeWindingTransformer`. We can make a new PR to update the API and add the `splitShuntAdmittance` parameter.

### CODE REVIEW

File: sensitivity-api/src/test/resources/SensitivityComputationParametersError.json [hunk 1]
```diff
@@ -1,12 +0,0 @@
-{
-  "version" : "1.0",
```
Comment (annetill) [inline sensitivity-api/src/test/resources/SensitivityComputationParametersError.json:2]: It is normal that you have delete this file?

Comment (marqueslanauja) [reply sensitivity-api/src/test/resources/SensitivityComputationParametersError.json:2]: I found that it is not used.

File: loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java [hunk 2]
```diff
@@ -84,14 +85,15 @@ protected static void load(LoadFlowParameters parameters, PlatformConfig platfor
...
-                    parameters.setT2wtSplitShuntAdmittance(config.getBooleanProperty("t2wtSplitShuntAdmittance", config.getBooleanProperty("specificCompatibility", DEFAULT_T2WT_SPLIT_SHUNT_ADMITTANCE)));
-                });
+            .ifPresent(config -> {
+                parameters.setVoltageInitMode(config.getEnumProperty("voltageInitMode", VoltageInitMode.class, DEFAULT_VOLTAGE_INIT_MODE));
+                parameters.setTransformerVoltageControlOn(config.getBooleanProperty("transformerVoltageControlOn", DEFAULT_TRANSFORMER_VOLTAGE_CONTROL_ON));
+                parameters.setNoGeneratorReactiveLimits(config.getBooleanProperty("noGeneratorReactiveLimits", DEFAULT_NO_GENERATOR_REACTIVE_LIMITS));
+                parameters.setPhaseShifterRegulationOn(config.getBooleanProperty("phaseShifterRegulationOn", DEFAULT_PHASE_SHIFTER_REGULATION_ON));
+                // keep old tag name "specificCompatibility" for compatibility
+                parameters.setT2wtSplitShuntAdmittance(config.getBooleanProperty("t2wtSplitShuntAdmittance", config.getBooleanProperty("specificCompatibility", DEFAULT_T2WT_SPLIT_SHUNT_ADMITTANCE)));
```
Comment (annetill) [inline loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java:94]: I understand that it is more coherent with our IIDM modeling to have 2 parameters to mention that the shunt as to be split, one for T2W and one for T3W. But for the powerflow engines I know, these two parameters should be identical (false for both or true for both). So I am not sure, I need to ask my colleagues.

Comment (annetill) [reply loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java:94]: It is maybe dangerous to have 2 parameters as in the use cases I have in mind these two parameters should be equal. Do you have in mind a use case where these two parameters could be different ?

Comment (marqueslanauja) [reply loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java:94]: No, but I will review the IOP cases to check it

**Comment by marqueslanauja** [comment]
Migration Guide 3.4.0 has been updated.

**Pull request merged**